packages/french-determiners/src/index.ts

```diff
@@ -23,6 +23,20 @@ type getDetParameters = {
   forceDes?: boolean;
 };
 
+/* 
+  ma, ta, sa deviennent mon, ton, son devant des noms féminins commençant par une voyelle pour éviter le hiatus de deux voyelles. 
+  a priori, ce n'est pas le cas avec y: par exemple ma yogourtière
+  https://open.byu.edu/grammaire_ouverte/determinants#:~:text=N'oubliez%20pas%20de%20faire,le%20hiatus%20de%20deux%20voyelles.
+*/
+function getFixHiatus(contentAfterDet: string) {

```

Comment:
suggest to rename in `mustFixHiatus` if responding with a boolean?
a function called `getFixHiatus` should take 3 parameters: contentAfterDet, classicOption (e.g. ma), hiatusOption (e.g. mon)

Reply:
done